CHANGES

```diff
@@ -120,6 +115,123 @@ Tools
   Verbose Compat vernacular, since these warnings can now be silenced or
   turned into errors using "-w".
 
+Bugfixes
+
```

Comment:
> Maybe I should simply write a last item saying that there are other fixes not mentioned here.

Seems reasonable.

However, more generally, what is your intent with the list of bugs? I mean, the list of bug fixes (and actually also of enhancements) is a very valuable information, but a request to the bug-tracker may be as much informative while being at the same time more customizable.

For instance, [a search on bugs with the FIXED tag and resolution date after Sep 2015](https://coq.inria.fr/bugs/buglist.cgi?f1=resolution&list_id=234517&o1=changedafter&order=Bug%20Number&query_format=advanced&resolution=FIXED&v1=2015-9-01) gives 424 bugs which altogether seem to include the list that you gave (and which goes closer to the 450 bugs counted by Matthieu).

So maybe the list could be replaced by a link to some well-chosen request listing the bugs, or by a copy-paste of the result of this request?